index.html

```diff
@@ -22,8 +25,132 @@ <h1 class="name" >Emilia Zambrano</h1>
+            <img class="imgArticle" src="proj_5.png" alt="First Project">
+            <article class="info">
+                <h2 class="proj-name">Project name goes here</h2>
+                <ul class="box">
+                    <li class="work-list">HTML/CSS</li>
+                    <li class="work-list">Ruby on Rails</li>
+                    <li class="work-list">JavaScript</li>      
+                </ul>
+                <button class="see" type="button">See this project <img src="arrow.png" alt="arrow"></button>
+            </article>     
+        </article>
+
+        <article class="work">
+            <img class="imgArticle" src="proj_6.png" alt="First Project">
+            <article class="info">
+                <h2 class="proj-name">Project name goes here</h2>
+                <ul class="box">
+                    <li class="work-list">HTML/CSS</li>
+                    <li class="work-list">Ruby on Rails</li>
+                    <li class="work-list">J vaScript</li>      
+                </ul>
+                <button class="see" type="button">See this project <img src="arrow.png" alt="arrow"></button>
+            </article>     
+        </article>
+
+    </section>
+
+    <section class="about">
+        <div class="text">
+            <h1 class="h1about">About</h1>
+            <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ea eiusmod</p>
+            <button class="cv" type="button">Get my CV <img src="download.png" alt="download"> </button>
+        </div>
+        <article class="languages">
+            <h3>Languages</h3>
+            <ul class="lang-list">
+                <li class= "about-list">JavaScript(ES5/ES6)</li>
+                <li class= "about-list">Ruby</li>
+                <li class= "about-list">HTML</li>
+                <li class= "about-list">CSS</li>
+            </ul>
+            <img class="hash" src="hash.png" alt="">
+        </article>
+
+        <article class="framework">
+            <h3>Frameworks</h3>
+            <ul class="lang-list">
+                <li class= "about-list">Bootstrap</li>
+                <li class= "about-list">Ruby on rails</li>
+                <li class= "about-list">RSpec</li>
+                <li class= "about-list">Capibara</li>
+                <li class= "about-list">Selenium</li>
+            </ul>
+            <img class="sq" src="fram.png" alt="">
+        </article>
+
+        <article class="skills">
+            <h3>Skills</h3>
+            <ul class="lang-list">
+                <li class= "about-list">Codekit</li>
+                <li class= "about-list">Codepen</li>
+                <li class= "about-list">GitAb</li>
+                <li class= "about-list">Terminal</li>
+            </ul>
+            <img class="bulb" src="skills.png" alt="">
+        </article>
```

Comment:
- [ ] [OPTIONAL] Very good using of `article` tag, and you can group all the 3 articles inside `ul` and `li`.